main/org.thingpedia.iot.humidity/eval/dev/annotated.txt

```diff
@@ -0,0 +1,20 @@
+====
+# online/ep39
+U: what's the humidity in the room
+UT: $dialogue @org.thingpedia.dialogue.transaction.execute;
+UT: now => @org.thingpedia.iot.humidity.humidity() => notify;
+====
+# online/123033
+U: what's the humidity on my thermostat
+UT: $dialogue @org.thingpedia.dialogue.transaction.execute;
+UT: now => @org.thingpedia.iot.humidity.humidity() => notify;
+====
+# online/62110256
+U: what's the humidity in the room
```

Comment:
This sentence looks like a duplicate? We should remove duplicates.

Reply:
ok